Cart/src/main/java/com/edu/dev/produtos/Producto.java

```diff
@@ -1,7 +1,10 @@
 package com.edu.dev.produtos;
 
 public class Producto {
+
 	private String codigo;
+	private double precio;
+	public static double IVA19 = 19;  
 
```

Comment:
Falto agregar constructor, así sea vacio

Reply:
Hola por qué es necesario?, gracias!

Reply:
SonarLint dice: Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation.